src/objects/Funding_Request__c.object

```diff
@@ -197,6 +197,14 @@
             </valueSetDefinition>
         </valueSet>
     </fields>
+    <fields>
```

Comment:
Did we make a decision internally about whether the field should be an external ID?